hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1545,7 +1533,7 @@ void lower(const VLS& e, movtdb& i, Vlabel b, size_t z) {
   lower_impl(e.unit, b, z, [&] (Vout& v) {
     auto d = v.makeReg();
     v << copy{i.s, d};
-    v << movtqb{d, i.d};
+    v << copy{d, i.d};
```

Comment:
@mxw - The code generator wouldn't accept it without the extra copy.  Something to do with changing register classes I suspect.